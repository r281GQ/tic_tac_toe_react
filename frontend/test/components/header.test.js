import React from 'react';
import { shallow } from 'enzyme';
import Header from './../../src/components/header';

describe('Header test', () => {
  it('should render', () => {
    const component = shallow(<Header createNewGameHandler={value => value} />);
    expect(component).toHaveLength(1);
  });
});
