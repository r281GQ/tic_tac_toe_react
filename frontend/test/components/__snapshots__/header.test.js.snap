// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test should render 1`] = `
<AppBar
  className="header"
  iconElementLeft={<GithubButton />}
  iconElementRight={
    <NewGameButton
      createNewGameHandler={[Function]}
    />
  }
  showMenuIconButton={true}
  title="Tic Tac Toe"
  zDepth={1}
/>
`;
